isc_joy/mappings/logitech.yaml

```diff
@@ -0,0 +1,63 @@
+# a blank mapping to serve as a template for new configurations
+# Type refers to whether a control is a Button (0) or an Axis (1). This determines what array the value comes from
+# Index refers to the position of the control in the joy message arrays
+
+mappings:
+ A:
+  type: 'button'
+  index: 1
+ B:
+  type: 'button'
+  index: 2
+ X:
+  type: 'button'
+  index: 0
+ Y:
+  type: 'button'
+  index: 3
+ LB:
+  type: 'button'
+  index: 4
+ RB:
+  type: 'button'
+  index: 5
+ BACK:
+  type: 'button'
+  index: 8
+ START:
+  type: 'button'
+  index: 9
+ GUIDE:
+  type: 'button'
+  index: -1
+ LS:
+  type: 'button'
+  index: 10
+ RS:
+  type: 'button'
+  index: 11
+ L_LR:
+  type: 'axis'
+  index: 0
+ L_UD:
+  type: 'axis'
+  index: 1
+ LT:
+  type: 'button'
```

Comment:
The xinput.msg is designed to send triggers as floats (to support analog triggers). Does your code support translating the binary button to +1.0 and -1.0?

Reply:
Yes, it's just an implicit cast from int to float. Same goes for the reverse

Reply:
Wouldn't that map pressed (true) to +1.0 and not pressed (false) to 0.0? Not pressed needs to be -1.0 (unless I have those backwards; that need to be double-checked). I don't see the code where this would happen.

Reply:
I just double checked. I had it backwards. -1.0 is fully pressed and 1.0 is not pressed.

Reply:
Solution is to use physical switch on gamepad.